src/stim/stabilizers/tableau_pybind_test.py

```diff
@@ -56,6 +54,27 @@ def test_from_named_gate():
     with pytest.raises(IndexError, match="not unitary"):
         stim.Tableau.from_named_gate("X_ERROR")
 
+def test_from_state_vector():
+    t = stim.Tableau.from_state_vector([
+        0.5**0.5,
+        0,
+        0,
+        0.5**0.5,
+    ], endian='little')
+    assert len(t) == 2
+    assert t.x_output(0) == stim.PauliString("Z_")
+    assert t.x_output(1) == stim.PauliString("_X")
+    assert t.z_output(0) == stim.PauliString("XX")
+    assert t.z_output(1) == stim.PauliString("ZZ")
+
+    unnormalized = stim.Tableau.from_state_vector([
```

Comment:
Also test cases that aren't nice numbers, and with intentional small floating point error.

Reply:
Added fuzz test to C++ layer.